packages/bottom-tabs/src/views/BottomTabItem.tsx

```diff
@@ -120,28 +120,29 @@ type Props = {
 };
 
 export default function BottomTabBarItem({
```

Comment:
As we are in here: please rename `BottomTabBarItem` -> `BottomTabItem` to make it consistent with the rest of the codebase. In code, it's used as `BottomTabItem` same as the name of the file.